experiments/process_sandbox/platform/sandbox.h

```diff
@@ -0,0 +1,40 @@
+// Copyright Microsoft and Project Verona Contributors.
+// SPDX-License-Identifier: MIT
+#pragma once
+
+/**
+ * This file contains the definition of a per-platform sandboxing policy.
+ * Because this interface currently has only one implementation, it is almost
+ * certainly the wrong abstraction and will change when others (e.g.
+ * seccomp-bpf) are added.
```

Comment:
outdated comment. Also, add the nice interface comments other common headers have.